modules/ROOT/pages/authentication-authorization/property-based-access-control.adoc

```diff
@@ -0,0 +1,73 @@
+:description: How to use Cypher to manage property-based access control on graphs.
+
+////
+[source, cypher, role=test-setup]
+----
+CREATE ROLE regularUsers;
+----
+////
+
+[role=enterprise-edition aura-db-enterprise]
+[[property-based-access-control]]
+= Property-based access control
```

Comment:
I think since it's an extension of the read privileges I'd put it after the read but before the write
```
*** xref:authentication-authorization/manage-privileges.adoc[]
*** xref:authentication-authorization/privileges-reads.adoc[]
*** xref:authentication-authorization/property-based-access-control.adoc[]
```